src/common/options.cc

```diff
@@ -7683,6 +7683,10 @@ static std::vector<Option> get_rbd_options() {
     .set_min(1)
     .set_description("minimum schedule tick (in milliseconds) for QoS"),
 
+    Option("rbd_qos_exclude_ops", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
+    .set_default(0)
+    .set_description("qos exclude ops, read = 1, write = 2, discard = 4, write_same = 8, compare_and_write = 16"),
```

Comment:
Nit: instead of a bitwise operator, can you switch this to a set of comma-delimated keys? See `rbd_default_features` for an example of how to attach a validator.